master/internal/kubernetes/pods.go

```diff
@@ -421,6 +421,7 @@ func (p *pods) summarize(ctx *actor.Context) map[string]agent.AgentSummary {
 			RegisteredTime: node.ObjectMeta.CreationTimestamp.Time,
 			Slots:          slotsSummary,
 			NumContainers:  len(podByNode[node.Name]),
+			ResourcePool:	"N/A",
```

Comment:
non-blocking: You can just name it to be an empty string since the name of a resource pool shouldn't be empty.

Reply:
Fixed